File: api/src/main/java/com/hirrao/javaee/entity/BloodPressure.java
package com.hirrao.javaee.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@TableName("bloodPressureLog")
public class BloodPressure {
    @TableId(value = "bplId")
    private Long bplId;
    private Long uid;
    @TableField("SBP")
    private Float SBP;
    @TableField("DBP")
    private Float DBP;
    @TableField("recordTime")
    private LocalDate recordTime;
    @TableField("riskLevel")
    private String riskLevel;
    private String classification;
}

File: api/src/main/kotlin/cn/hirrao/javaee/entity/BloodPressure.kt
package cn.hirrao.javaee.entity

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import java.time.LocalDate

@TableName("bloodPressureLog")
data class BloodPressure(
    @TableId(value = "bplId") val bplId: Long? = null,
    val uid: Long? = null,
    @TableField("SBP") val SBP: Float? = null,
    @TableField("DBP") val DBP: Float? = null,

    @TableField("recordTime") val recordTime: LocalDate? = null,

    @TableField("riskLevel") val riskLevel: String? = null,
    val classification: String? = null
)